api/services/userService.js

```diff
@@ -27,6 +34,39 @@ const signUp = async (name, email, password, address, phoneNumber) => {
     return await userDao.createUser(name, email, hashedPassword, address, phoneNumber)
 }
 
+const signIn = async (email, password) => {
+    validator.validateEmail(email);
+    validator.validatePassword(password);
+
+    const user = await userDao.getUserByEmail(email);
+
+    if (!user) {
+        const error = new Error('User not found');
```

Comment:
로그인 에러 처리 시에는 ID가 틀렸는지 Password가 틀렸는지 모르게 하는 것이 좋습니다.

```suggestion
        const error = new Error('Invalid user');
```